src/chromap.h

```diff
@@ -18,7 +18,7 @@
 #include "sequence_batch.h"
 #include "temp_mapping.h"
 
-#define CHROMAP_VERSION "0.1.4-r284"
+#define CHROMAP_VERSION "0.1.3-r257"
```

Comment:
Why change the version?

Reply:
Fixed. Not sure why this happened, maybe because the rebase of this old branch had some issues.